transformations/longer_names_ner/test.json

```diff
@@ -1,137 +1,29 @@
 {
-  "type": "butter_fingers_perturbation",
+  "type": "longer_names_ner",
   "test_cases": [
     {
-      "input_sequence": [
-        "Manmohan",
-        "Singh",
-        "served",
-        "as",
-        "the",
-        "PM",
-        "of",
-        "India",
-        "."
-      ],
-      "input_tag": [
-        "B-PER",
-        "I-PER",
-        "O",
-        "O",
-        "O",
-        "O",
-        "O",
-        "B-COUNTRY",
-        "O"
-      ],
-      "output_sequence": [
-        "Manmohan",
-        "M",
-        "Singh",
-        "served",
-        "as",
-        "the",
-        "PM",
-        "of",
-        "India",
-        "."
-      ],
-      "output_tag": [
-        "B-PER",
-        "I-PER",
-        "I-PER",
-        "O",
-        "O",
-        "O",
-        "O",
-        "O",
-        "B-COUNTRY",
-        "O"
-      ]
+      "input_sequence": "Manmohan Singh served as the PM of India .",
+      "output_sequence": "Manmohan M Singh served as the PM of India .",
+      "input_tag": "B-PER I-PER O O O O O B-COUNTRY O",
+      "output_tag": "B-PER I-PER I-PER O O O O O B-COUNTRY O"
     },
     {
-      "input_sequence": [
-        "Neil",
-        "Alden",
-        "Armstrong",
-        "was",
-        "an",
-        "American",
-        "astronaut"
-      ],
-      "input_tag": [
-        "B-PER",
-        "I-PER",
-        "I-PER",
-        "O",
-        "O",
-        "B-COUNTRY",
-        "B-COUNTRY"
-      ],
-      "output_sequence": [
-        "Neil",
-        "Y",
-        "Alden",
-        "Armstrong",
-        "was",
-        "an",
-        "American",
-        "astronaut"
-      ],
-      "output_tag": [
-        "B-PER",
-        "I-PER",
-        "I-PER",
-        "I-PER",
-        "O",
-        "O",
-        "B-COUNTRY",
-        "B-COUNTRY"
-      ]
+      "input_sequence": "Neil Alden Armstrong was an American astronaut",
+      "output_sequence": "Neil Y Alden Armstrong was an American astronaut",
+      "input_tag": "B-PER I-PER I-PER O O B-COUNTRY B-COUNTRY",
```

Comment:
Input tag should be: "B-PER I-PER **B**-PER O O B-COUNTRY B-COUNTRY".
Can we double-check this?

Reply:
No.. the tags are fine..